packages/components/field-label/src/field-label.tsx

```diff
@@ -130,13 +146,17 @@ const FieldLabel = (props: TFieldLabelProps) => {
               </Inline>
             )}
             {props.hint && (
-              <Text.Detail tone={props.tone}>{props.hint}</Text.Detail>
+              <Text.Detail tone={getInfoTextTone(theme, props.tone)}>
```

Comment:
Nit: I would keep using a ternary here, to keep it consistent with other similar cases in this file.

Reply:
That's a good point! Thanks! [commit](https://github.com/commercetools/ui-kit/pull/2356/commits/3ae00e8a13eeead2db040e1408dcebf337455bcc)